packages/contracts/deploy/new/20_permissions/10_dao-registry-permissions.ts

```diff
@@ -3,17 +3,19 @@ import {DeployFunction} from 'hardhat-deploy/types';
 
 import {Operation} from '../../../utils/types';
 import {getContractAddress, managePermissions} from '../../helpers';
+import {DAO__factory} from '../../../typechain';
```

Comment:
Is it possible to set up a path, where it's clear that we're importing the current artifacts?

Reply:
yes we can, but if we do /typechain/current/ for example! 
that mean we need to update every single test file and correct the paths

i suggest creating a task only for this, ur call

Reply:
Sounds good, follow-up task created OS-430